Composer/packages/client/src/pages/publish/type.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Where is there a mixture of types and interfaces? please use types all across.

Reply:
rename